lib/ex_public_key/ex_rsa_public_key.ex

```diff
@@ -68,6 +68,40 @@ defmodule ExPublicKey.RSAPublicKey do
     |> from_der_encoded_0()
   end
 
+  # Protocols
+
+  defimpl Inspect do
+    import Inspect.Algebra
+
+    @doc """
+    Formats the RSAPrivateKey and includes the SHA256 fingerprint.
+
+    example:
+    ```
+    #ExPublicKey.RSAPublicKey<
```

Comment:
Neat! I'd personally set nest to 2, as the one-space indent triggers me, but otherwise this looks good.

Reply:
I can set nest to 2, I was trying to follow the pattern for nesting in the elixir source code, but it appears I was looking at an older version of the code. In the version I was looking at nest always seemed to be 1, but looking at the latest code it seems they've changed it to 2, so I'll updated this to follow the same pattern :)